build.gradle

```diff
@@ -62,11 +62,11 @@ sourceSets {
 
 sharedLibrary {
     coreVersion = '2.387.1' // https://repo.jenkins-ci.org/public/org/jenkins-ci/main/jenkins-core/
-    testHarnessVersion = '2002.v0b_78b_a_d69e5d' // https://mvnrepository.com/artifact/org.jenkins-ci.main/jenkins-test-harness?repo=jenkins-releases
+    testHarnessVersion = '2034.v41c9cb_349299' // https://mvnrepository.com/artifact/org.jenkins-ci.main/jenkins-test-harness?repo=jenkins-releases
     pluginDependencies {
-        workflowCpsGlobalLibraryPluginVersion = '570.v21311f4951f8' // https://repo.jenkins-ci.org/public/org/jenkins-ci/plugins/workflow/workflow-cps-global-lib/
+        workflowCpsGlobalLibraryPluginVersion = '609.vd95673f149b_b' // https://repo.jenkins-ci.org/public/org/jenkins-ci/plugins/workflow/workflow-cps-global-lib/
```

Comment:
Looks like mend is asking to upgrade to next version https://github.com/opensearch-project/opensearch-build-libraries/pull/290/checks?check_run_id=16085684925

Reply:
Yeah Sayali, I'm checking out some other dependencies too.